<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackgroundSubtractionStepDialog</class>
 <widget class="QDialog" name="BackgroundSubtractionStepDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BackgroundSubtractionStepDialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_learningrate">
      <property name="font">
       <font>
        <family>Replica-Bold</family>
        <pointsize>15</pointsize>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>Lerningrate</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSlider" name="slider_learningrate">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>-1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber">
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
          <property name="value" stdset="0">
           <double>-1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <item>
         <property name="text">
          <string>FGD_STAT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MOG</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MOG2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GMG</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
